docs/running-on-kubernetes.md

```diff
@@ -889,7 +889,10 @@ See the [configuration page](configuration.html) for information on Spark config
   <td><code>spark.kubernetes.executor.podNamePrefix</code></td>
   <td>(none)</td>
   <td>
-    Prefix to use in front of the executor pod names.
+    Prefix to use in front of the executor pod names. It must conform the rules defined by the Kubernetes
+    <a href="https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-label-names">DNS Label Names</a>.
+    Besides, the executor pod names will be generated in the form of <code>$podNamePrefix-exec-$id</code>, where the `id` is
```

Comment:
I'm not sure I understand what "besides" is reference here, could you clarify this part?

Reply:
This part is to notify users that spark uses this config to generate pod names internally. the max length will be limited by both `podNamePrefix` and the suffix `-exec-$id`

Reply:
Could we perhaps rephrase this as:

    Prefix to use in front of the executor pod names. It must conform the rules defined by the Kubernetes
    <a href="https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-label-names">DNS Label Names</a>.
The prefix will be used to generate executor pod names in the form of <code>$podNamePrefix-exec-$id</code>, where the `id` is